app/src/renderer/components/monitor/PageValidatorSlashes.vue

```diff
@@ -1,10 +1,12 @@
 <template lang="pug">
 page(icon="storage" :title="`${validatorId} Slashes`")
-  tool-bar
-    router-link(
-      :to="{ name: 'validator', params: { validator: $route.params.validator }}")
+  div(slot="menu"): tool-bar
+    router-link(:to="{ name: 'validator', params: { validator: $route.params.validator }}")
       i.material-icons arrow_back
-    a(@click='toggleSearch'): i.material-icons search
+      .label Back
+    a(@click='toggleSearch')
+      i.material-icons search
+      .label Search
 
   part(title='Statistics')
```

Comment:
dd="2 weeks ago%" <- percent sign (not important right now)

Reply:
+ list items link to nowhere

Reply:
Fixed the % issue.